guides/common/modules/proc_changing-email-notification-settings-for-a-host.adoc

```diff
@@ -5,12 +5,12 @@
 You can configure {Project} to send email notifications either to an individual user or a user group.
 When set to a user group, all group members who are subscribed to the email type receive a message.
 
-To view the notification status for a host, navigate to *Hosts* > *All Hosts* and click the host you want to view.
-In the host details page, click the *Additional Information* tab, you can view the email notification status. 
-
 Receiving email notifications for a host can be useful, but also overwhelming if you are expecting to receive frequent errors, for example, because of a known issue or error you are working around.
-To change the email notification settings for a host, complete the following steps.
+To view or change the email notification settings for a host, complete the following steps.
```

Comment:
```suggestion
```

I would just drop this sentence. The file is called "Changing Email Notification Settings for a Host" and the next heading is "Procedure", so I believe we can do without.